kiwi/bootloader/config/zipl.py

```diff
@@ -101,9 +101,15 @@ def set_loader_entry(self, root_dir: str, target: str) -> None:
                 f'Kernel lookup in {root_dir}/lib/modules failed with {issue}'
             )
         default_entry = f'{os_release.get("ID")}-{kernel_name}.conf'
+        # Check if there is already an entry file name produced by the
+        # distribution logic/policy. If we can find an already present
+        # entry name, prefer this name over the kiwi naming policy
+        for dist_entry in glob.iglob(f'{root_dir}/{self.entries_dir}/*'):
+            default_entry = os.path.basename(dist_entry)
+            break
```

Comment:
Shouldn't this be in bootloader_spec_base for all bls bootloaders to use?

Reply:
good point, yes

Reply:
done